torch_ort_inference/setup.py

```diff
@@ -0,0 +1,49 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import setuptools
+from datetime import date
+import os
+
+with open("pypi-readme.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+def fetch_requirements(path):
```

Comment:
Are we going to use the extra to install openvino torch_ort_inference? I thought we wanted to add support for `pip install torch_ort_inference[openvino]`

Reply:
Added changes.